README.md

```diff
@@ -34,7 +34,7 @@ The following list describes the main folders in this repository.
   Code)
 - `assets` - contains downloadable files linked in the documentation
 - `reference` - contains the documentation published to
-  [docs.microsoft.com]([https://docs.microsoft.com/powershell/scripting/). This includes both
+  [PowerShell technical documentation](/powershell/scripting/). This includes both
```

Comment:
The README is not published to Microsoft Docs so the URL shouldn't be changed. It must be fully-qualified to work in GitHub.